Android FileProvider Toast showing issue resolved

sandeepdillerao opened the pull request.

sandeepdillerao committed:
FileProvider for android 7+
Showing Toast is removed

sandeepdillerao committed:
FileProvider for android 7+
Showing Toast is removed

sandeepdillerao committed:
FileProvider for android 7+
Showing Toast is removed

sandeepdillerao committed:
FileProvider for android 7+
Showing Toast is removed

sandeepdillerao committed:
FileProvider for android 7+
Showing Toast is removed

abomadi commented:
@sandeepdillerao i have used ur branch, however using Hotline.showFAQs(); on Android crashes, Android version 5.0 hotline 1.2.1 any idea ?

abomadi commented:
@sandeepdillerao i have used ur branch, however using Hotline.showFAQs(); on Android crashes, Android version 5.0 hotline 1.2.1 any idea ?

sandeepdillerao commented:
i have not made any changes in FAQ.


*Regards,Sandeep Dillerao*
*Mob.: +91 8483094292*
*Mail : sandydillerao@gmail.com <sandeep.dillerao@tcs.com>*

On 24 October 2017 at 17:09, abomadi <notifications@github.com> wrote:

> @sandeepdillerao <https://github.com/sandeepdillerao> i have used ur
> branch, however using Hotline.showFAQs(); on Android crashes, Android
> version 5.0 hotline 1.2.1 any idea ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/freshdesk/hotline-phonegap/pull/35#issuecomment-338962050>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF-dhdh-72H5Td3BYXbMGwqCivKVP7ywks5svcxvgaJpZM4OSCas>
> .
>

sandeepdillerao committed:
app v7 support Library issue for android lolipop fix
E/UncaughtException: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/freshdesk/hotline/activity/InterstitialActivity;

abomadi commented:
@sandeepdillerao i am not able to use your branch via npm ! @sachinfreshdesk why dont u guys merge his branch

sandeepdillerao commented:
Install via github url it will work

Regards,
Sandeep Dillerao
Software Engineer
+91 8483094292
+91 7013842232

On 14-Dec-2017 4:36 PM, "Abd Madi" <notifications@github.com> wrote:

> @sandeepdillerao <https://github.com/sandeepdillerao> i am not able to
> use your branch via npm ! @sachinfreshdesk
> <https://github.com/sachinfreshdesk> why dont u guys merge his branch
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/freshdesk/hotline-phonegap/pull/35#issuecomment-351680259>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF-dhQP24LzAc5YDx5PqrzxGFW7pIYuSks5tAQE_gaJpZM4OSCas>
> .
>

sandeepdillerao committed:
plugin.xml package_name change to applicationId

sandeepdillerao committed:
play service version code set to 11.6.2
fileProvider removed

sandeepdillerao committed:
file provider conflict with camera and file provider plugin resolved
